### PR TITLE
Mobile Version Skeleton: 

### PR DESCRIPTION
**Pull request by Enning94**
## Mobile Version Skeleton of the Figma portfolio template 2

In this project I:

-Added the toolbar to the personal portfolio project
-Added the headline to the personal portfolio project
-Added markup and styling

### CODE REVIEW

**Review by aboongm — CHANGES_REQUESTED** [review]
Hi @Enning94 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

request_changes_1

## Highlights
- All linters are passing ✔️
- Good commit history ✔️
- Good PR description ✔️

## Required Changes ♻️

- Your .gitignore file should ignore the node_modules folder. You should include "node_modules" in .gitignore file
- for other required changes, please check the comments below.


## Optional suggestions

[OPTIONAL] 
- Include the linters file eg .hintrc, .stylelintrc and .eslintrc
- Include the link to MIT license


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    
+  </head>
+  <body>
+    <main>
+    <header class="container">
+      <a href="#" class="logo"><img src="./Logo_placeholder.png" alt="Logo"/></a>
+      <a href="#" class="menu"><img src="./Enabled.png" alt="Menu button"/></a>
```
Comment (youmari) [inline index.html:18]: - [ ] nice implementing the requests by the previous reviewer however you should keep your `<nav>` tag so you should wrap you hamburger button with the `<nav>` tag cause it's part of your navigations

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Enning94 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Good commit messages
✔️ No linter error
✔️ Implementing the requirements

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <main>
+      <header>
+        <nav class="container">
+          <a href="#" class="logo"
+            ><img src="./Logo_placeholder.png" alt="Logo"
+          /></a>
+          <a href="#" class="menu"
+            ><img src="./Enabled.png" alt="Menu button"
+          /></a>
+        </nav>
```
Comment (danmainah) [inline index.html:26]: - [ ] @Enning94 great job implementing the changes, kindly ensure only the hamburger menu is covered by the `nav` tag, the logo is outside the nav tag.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Enning94 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <a href="#" class="twitter"
+              ><img src="./Twitter.png" alt="Twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="vector"
+              ><img src="./Vector.png" alt="Vector logo"
+            /></a>
+          </li>
+        </ul>
```
Comment (ishimwezachee) [inline index.html:58]: - [ OPTIONAL] you've done a great job implementing the changes from the previous reviewer, but I would suggest fixing this icon links. 
image

**Review by ishimwezachee — APPROVED** [review]
Hi @Enning94 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
All linters are passing ✔️
Good commit history ✔️
Good PR description ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**